### DISCUSSION TITLE
Error on 2k Indoor Camera

### DISCUSSION THREAD

**Discussion by AMG5TAG** [samemory/homebridge-eufy-security-control#381] [Q&A]
I've setup the plugin as describer but getting this error on my camera

[4/4/2022, 4:42:54 PM] [EufySecurity] FFmpeg process creation failed: spawn ffmpeg ENOENT Finns Room

Im using my current 2K indoor camera as testing to see if this plugin works well before diving in with more cameras. 

Also when viewing in Homekit is it 2K or 1080p?

**Comment by schliemann** [comment]
It means the plugin can't find ffmpeg.

**Reply by AMG5TAG** [reply]
How do I go about fixing that, I have my server running on MacOS if this helps

**Reply by lenoxys** [reply]
Is it high sierra version ? We don't support since homebridge team doesn't support it

**Reply by AMG5TAG** [reply]
No it's a M1 Mac mini running Monterey

**Reply by AMG5TAG** [reply]
I apologies, I didn't have the correct ffmpeg installed, which I've now rectified but now I'm getting the following error.

[4/5/2022, 2:40:05 PM] [EufySecurity-1.0.8] ERROR: FFmpeg exited with code: null and signal: SIGABRT (Unexpected) Finns Room

This is the new error I keep getting

**Reply by lenoxys** [reply]
Can you fill an issue thought GitHub this is discussion section.
Thanks !
